### PR TITLE
site: update demo Tabs usage

### PR DESCRIPTION
**Pull request by li-jia-nan**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | 无需纳入ChangeLog |
| 🇨🇳 Chinese | 无需纳入ChangeLog |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Review by kiner-tang — APPROVED** [review]

File: .dumi/theme/builtins/InstallDependencies/index.tsx [hunk 1]
```diff
@@ -13,68 +13,31 @@ interface InstallProps {
...
-          ),
-        },
-      ].filter((item) => item.children),
-    [npm, yarn, pnpm],
-  );
+  const items: Tab[] = [
+    {
+      key: 'npm',
+      label: 'npm',
+      children: npm ? <SourceCode lang="bash">{npm}</SourceCode> : null,
```
Comment (crazyair) [inline .dumi/theme/builtins/InstallDependencies/index.tsx:22]: 这个之前为什么写三元，而不是直接 chilren: <SourceCode lang="bash">{npm}</SourceCode>

File: .dumi/theme/builtins/InstallDependencies/npm.tsx [hunk 2]
```diff
@@ -1,26 +1,39 @@
 import React from 'react';
+import { createStyles, css } from 'antd-style';
+import classNames from 'classnames';
 
 interface IconProps {
   className?: string;
   style?: React.CSSProperties;
 }
 
+const useStyle = createStyles(() => ({
```
Comment (crazyair) [inline .dumi/theme/builtins/InstallDependencies/npm.tsx:10]: 这个从原来的一次，改成了3次？

File: .dumi/theme/builtins/InstallDependencies/index.tsx (hunk 1 above)
Comment (li-jia-nan) [reply .dumi/theme/builtins/InstallDependencies/index.tsx:22]: 后面有一个 `filter((item) => item.children)` 过滤的逻辑，需要把 children 为空的项排除掉，如果按照你的写法，children 一定不会为空了

File: .dumi/theme/builtins/InstallDependencies/npm.tsx (hunk 2 above)
Comment (li-jia-nan) [reply .dumi/theme/builtins/InstallDependencies/npm.tsx:10]: 不是一个逻辑，这里是在 svg 外面包了一层 span，为了和 antd-icon 的规范对齐

File: .dumi/theme/builtins/InstallDependencies/index.tsx [hunk 3]
```diff
@@ -13,68 +13,31 @@ interface InstallProps {
...
-            <div className={styles.packageManager}>
-              <PnpmLogo />
-              <span>pnpm</span>
-            </div>
-          ),
-        },
-      ].filter((item) => item.children),
-    [npm, yarn, pnpm],
-  );
+  const items: Tab[] = [
```
Comment (crazyair) [inline .dumi/theme/builtins/InstallDependencies/index.tsx:18]: useMemo 给去掉了？

Comment (li-jia-nan) [reply .dumi/theme/builtins/InstallDependencies/index.tsx:18]: 对，没必要加，这里逻辑很简单，useMemo 属于是负优化了

**Pull request merged**